### ISSUE TITLE
config file is not working | fabric 1.19.2

### ISSUE THREAD

**Issue by Surpos0** [Jasel-Umena/Apocalypse-Origins#2]
I wanted to disable some elements of hexblade gameplay but, some of the configs are just doing nothing no matter what i change.
I mean this ones:
      "apocalypse:hexblade/description_soul_barrier": true,
      "apocalypse:hexblade/description_risk": true,
      "apocalypse:hexblade/description_nostalgia": true,
      "apocalypse:hexblade/description_buoyancy": true,
      "apocalypse:hexblade/description_one_soul": true,
      "apocalypse:hexblade/description_item_curse": false,
      "apocalypse:hexblade/description_smite_of_radiance": true,
      "apocalypse:hexblade/description_hunters": true,
      "apocalypse:hexblade/description_inanimate": false,
      "apocalypse:hexblade/description_intangible": true,
      "apocalypse:hexblade/description_timeless": true,
      "apocalypse:hexblade/description_dash_slash": true,
      "apocalypse:hexblade/description_parry": true,
      "apocalypse:hexblade/description_secondary": true

**Comment by Jasel-Umena** [comment]
You do realize you are literally disabling the description of the power and not the power itself, right?

**Comment by Surpos0** [comment]
Oooo... but that is dumb why this option is even there?

**Comment by Jasel-Umena** [comment]
The powers you are trying to turn off should be in hexblade/hexblade_passives_

**Comment by Surpos0** [comment]
but when i turn this off all other powers are disabled too

**Comment by Surpos0** [comment]
like its just makes you a normal human with dash

**Comment by Jasel-Umena** [comment]
Is there not an option to disable the sub-powers?

**Comment by Surpos0** [comment]
nope
 "apocalypse:hexblade": {
      "enabled": true,
      "apocalypse:hexblade/hexblade_callback": true,
      "apocalypse:hexblade/hexblade_passives": true,
      "apocalypse:hexblade/hexblade_resource_risk": true,
      "apocalypse:hexblade/hexblade_resource_nostalgia": true,
      "apocalypse:hexblade/hexblade_primary": true,
      "apocalypse:hexblade/hexblade_primary_b": true,
      "apocalypse:hexblade/hexblade_secondary": true,
      "apocalypse:hexblade/hexblade_right_click": true,
      "apocalypse:hexblade/hexblade_resource_ultimates": true,
      "apocalypse:hexblade/description_soul_barrier": false,
      "apocalypse:hexblade/description_risk": false,
      "apocalypse:hexblade/description_nostalgia": false,
      "apocalypse:hexblade/description_buoyancy": false,
      "apocalypse:hexblade/description_one_soul": false,
      "apocalypse:hexblade/description_item_curse": false,
      "apocalypse:hexblade/description_smite_of_radiance": false,
      "apocalypse:hexblade/description_hunters": false,
      "apocalypse:hexblade/description_inanimate": false,
      "apocalypse:hexblade/description_intangible": false,
      "apocalypse:hexblade/description_timeless": false,
      "apocalypse:hexblade/description_dash_slash": false,
      "apocalypse:hexblade/description_parry": false,
      "apocalypse:hexblade/description_secondary": false

**Comment by Jasel-Umena** [comment]
Well, in that case you can't toggle those aspects off.

**Comment by Surpos0** [comment]
like every other origin you made have config where you can turn on and off specific powers

**Comment by Jasel-Umena** [comment]
Yes, but the passives are all in the same power file, since they all work off each other and removing them would cause the origin to break.

**Comment by Jasel-Umena** [comment]
You can still edit the file to disable those aspects you don't like
